discovery_engine_core/ai/kpe/extract_params.py

```diff
@@ -29,14 +29,17 @@
 
 BYTE_ORDER: str = "little"
 
+
```

Comment:
Now sure why we are adding white space. We didn't discuss any style convention or formatting tool so we avoid to change this file more then necessary.

Reply:
good question, happened by mistake, not sure why it happened at all, it's a python file after all 🤔 i'll revert it.